templates/bin_setup.erb

```diff
@@ -8,7 +8,7 @@ set -e
 
 # Set up Ruby dependencies via Bundler
 gem install bundler --conservative
-bundle check || bundle install
+bundle check || bundle install --path=vendor/bundle
```

Comment:
You're right, this will affect local dev machines, as well. That's a side effect, and not necessary to fix travis. Ideas on how to implement the custom bundler path only on Travis?